pkg/monitortests/kubeapiserver/auditloganalyzer/audit_log_summarizer.go

```diff
@@ -3,15 +3,20 @@ package auditloganalyzer
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/openshift/origin/pkg/dataloader"
+	"github.com/sirupsen/logrus"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
 )
 
+var monitoredUsers = []string{"kube-", "openshift-", "system:node"}
```

Comment:
want to make these all prefixes?  `system:serviceaccount:kube-`, `system:serviceaccount:openshift-`, `system:node`.